bc/channel/utils/connectors/bluesky_api/client.py

```diff
@@ -229,6 +259,20 @@ def _parse_text_facets(self, text) -> list[TextAnnotation]:
                 ],
             }
             facets.append(annotation)
+
+        for u in self._parse_tags(text):
+            annotation = {
+                "index": {
+                    "byteStart": u.start,
+                    "byteEnd": u.end,
+                },
+                "features": [
+                    {
+                        "$type": "app.bsky.richtext.facet#tag",
+                        "tag": u.text,
+                    }
+                ],
+            }
```

Comment:
This code looks good, but we need to add the `annotation` to the facets list. You can achieve this by appending the `annotation` variable, `facets.append(annotation)`, like line 261

Reply:
Looks like I've forgotten to actually append it, thanks!